README.md

```diff
@@ -14,31 +14,32 @@
 [![DOI](https://zenodo.org/badge/316911228.svg)](https://zenodo.org/badge/latestdoi/316911228)
 
 ### Overview
-numba-mpi provides Python wrappers to the C MPI API callable from within [Numba JIT-compiled code](https://numba.readthedocs.io/en/stable/user/jit.html) (@jit mode). For an outline of the project, rationale, architecture, and features, refer to: [numba-mpi arXiv e-print](https://doi.org/10.48550/arXiv.2407.13712) (please cite if numba-mpi is used in your research).
+numba-mpi provides Python wrappers to the C MPI API callable from within [Numba JIT-compiled code](https://numba.readthedocs.io/en/stable/user/jit.html) (@jit mode). For an outline of the project, rationale, architecture, and features, refer to: [numba-mpi paper in SoftwareX (open access)](https://www.sciencedirect.com/science/article/pii/S235271102400267X) (please cite if numba-mpi is used in your research).
```

Comment:
that's seems to be an internal ScienceDirect URL, let's use https://doi.org/... here

Reply:
I tried that first but for me it does not redirect properly and leads to TIMEOUT. BTW, I encountered the same broken link issue somewhere else where You have used DOI-based link as well - please check it [here](https://doi.org/10.1016/j.softx.2024.101897) (maybe it's only issue on my side), then I'll replace it. And if its broader, I'd keep this internal link for the time being and report issue to SoftwareX.

Reply:
Ineed, it timesouts for me as well. Indeed, best to keep the sciencedirect link, and it's worth reporting to SoftwareX, thank you!